src/main/pages/che-7/administration-guide/assembly_using-organizations.adoc

```diff
@@ -0,0 +1,224 @@
+CHE_MAIL_HOST=smtp.gmail.com
+CHE_MAIL_SMTP_STARTTLS_ENABLE=true
+CHE_MAIL_SMTP_AUTH=true
+CHE_MAIL_SMTP_AUTH_USERNAME=no-reply@gmail.com
+CHE_MAIL_SMTP_AUTH_PASSWORD=password
+----
+
+
+[id="creating-sub-organizations"]
+=== Creating sub-organizations
+
+To create a sub-organization:
+
+*  On the *Organization Details* page, select the *Sub-Organizations* tab.
+
+*  Click the *Add Sub-Organization* button.
+
+
+The steps to create a sub-organization are the same as that for creating an organization. Use them to create the organization.
+
+[id="adding-members-to-sub-organizations"]
+=== Adding members to sub-organizations
+
+You can only add members of the parent organization as members of the sub-organization.
+
+
+[id="organization-and-sub-organization-administration"]
+=== Organization and sub-organization administration
+
+The settings of the organization are visible to all members of the organization.  Only the Eclipse Che system administrator can modify the settings.
+
+
+[id="renaming-an-organization-or-sub-organization"]
+=== Renaming an organization or sub-organization
+
+[NOTE]
+====
+Only an Eclipse Che system administrator and administrator of the organization can rename an organization or sub-organization.
+====
+
+To rename an organization:
+
+. Click the *Name* field to edit the name of the organization.  The save mode appears.
+
+. Click the *Save* button to update the name.
+
+The name of the organization or sub-organization must follow these rules: 
+
+* Only alphanumeric characters and a single dash (-) can be used. 
+
+* Spaces cannot be used. 
+
+* Each organization name must be unique within the Eclipse Che installation. 
+
+* Each sub-organization name must be unique within an organization.
+
+
+[id="leaving-an-organization-or-sub-organization"]
+=== Leaving an organization or sub-organization
+
+To leave an organization, members need to contact the administrator of the organization or the system administrator of Eclipse Che.  
+
+
+[id="deleting-an-organization-or-sub-organization"]
+=== Deleting an organization or sub-organization
+
+[IMPORTANT]
+====
+* Only system administrators or administrators of the organization can delete an organization or sub-organization.
+* This action cannot be reverted, and all workspaces created under the organization will be deleted.
+* All members of the organization will receive an email notification to inform them about the deletion of the organization.
+====
+
+To delete an organization or a sub-organization:
+
+* Click the *Delete* button.
+
+
+[id="allocating-resources-for-organizations"]
+=== Allocating resources for organizations
+
+Workspaces use the resources of the organization that are allocated by the system administrator. The resources for sub-organizations are taken from the parent organization. Administrators control the portion of resources, of the parent organization, that are available to the sub-organization.
+
+
+[id="managing-limits"]
+=== Managing limits
```

Comment:
We need to explain that we inherit from the limits configured on the underlying platform K8S or OpenShift
cc @skabashnyuk and @sleshchenko

Reply:
ack.